mono/metadata/unity-utils.c

```diff
@@ -374,6 +374,8 @@ MONO_API gboolean
 mono_class_is_blittable(MonoClass *klass)
 {
 	g_assert(klass);
+	if (!klass->fields_inited)
```

Comment:
@joncham should we also check if the class has been inited here or is just checking the fields fine?

Reply:
It looks like field initialization is enough.